### ISSUE TITLE
Feature request: Make waypoints in-game available with GPS in inventory only.

### ISSUE THREAD

**Issue by vinyljunkie** [DayZMod/DayZ#708]
It would be cool if waypoints were available when the player finds a GPS.

Thanks.

**Comment by oiad** [comment]
I like what you are trying to do here but my only thoughts are that it's a massive change from how it's worked in the past so players might not expect a change

**Comment by vinyljunkie** [comment]
It doesn't have to be a massive change. Foxy had an idea the waypoints and distance would only show in the GPS window. This way servers with Waypoints enabled won't notice any difference and servers without them will have, for players,  an added value to finding a GPS.

**Comment by AsFoxy** [comment]
Fixing game breaking bugs like duping and combat logging will be a massive change as well but that doesn't mean we won't gladly do it.

**Comment by bchawke** [comment]
I'd love to see this feature as well.  I've suggested having the markers show up only in the GPS window in the past as well.  It would add so much to the game and increase the value of the GPS for servers with map markers disabled (which in my opinion should be all vanilla servers).

**Comment by skigoggles** [comment]
I agree that this is logical, but the current behaviour has been there forever with just the map. If you want a server setting to push it to GPS (or map+compass), go for it. I don't see the value in disrupting the shrinking player base.  We need to add new features, not take anything away.

**Comment by vinyljunkie** [comment]
US 434 just removed their waypoints upon request from their players. Player numbers are increasing after the release of 1.8.8 and both 434 and Europa are top 10 worldwide as I am typing this. This would be a good time to add this.

**Comment by Sportivoman** [comment]
Agreed as have all the other admins from 434, would make the GPS useful ingame. We're the only two servers that have true Vanilla and are populated. of late we've been paying close attention to our playerbase inregards to changes.

**Comment by skigoggles** [comment]
make this an **_option_** if you have to have it.

**Comment by LO0TC1F3R** [comment]
can you remove the distance to the waypoint though? just so range finders keep their purpose.
